wikipedia-dark.user.css

```diff
@@ -559,6 +674,50 @@
   #mwe-popups-settings {
     box-shadow: 0 2px 2px 0 rgba(85, 85, 85, .25) !important;
   }
+  /* wikidata fixes */
+  .wikibase-entitytermsforlanguagelistview-header-row th {
+    background-color: #333 !important;
+  }
+  .wikibase-sitelinkgroupview, .ui-widget.ui-suggester-list {
+    background-color: #222 !important;
+  }
+  li.wikibase-sitelinkview.listview-item span.wikibase-sitelinkview-siteid-container {
+    background-color: #333 !important;
+  }
+  ul.wikibase-snaktypeselector-menu.ui-menu {
+    background-color: #222 !important;
+  }
+  .wikibase-snaktypeselector-menu .ui-state-active,
+  .wikibase-snaktypeselector-menu .ui-state-focus {
+    background: #333 !important;
+  }
+  .wb-icon, .articlepreview-button, .wb-help-field-hint {
+    filter: invert(80%);
+  }
+  .tipsy-inner {
+    color: #9a9a9a;
+    background-color: #222 !important;
+  }
+  .articlepreview.open {
+    background-color: #222 !important;
+  }
+  .articlepreview .toggler {
+    filter: invert(80%);
+    background-color: #eee !important;
+  }
+  .wikibase-entityview textarea {
+    color: #9a9a9a;
+  }
+  .imeselector-toggle {
+    filter: brightness(200%);
+  }
+  .imeselector-menu {
+    filter: brightness(50%);
+    background-color: #222 !important;
+  }
+  .imeselector-menu .selectable-row:hover {
+    background-color: #333;
+  }
```

Comment:
Again here above I see many selectors that need to be condensed into the same property rule.

Reply:
Yes a lot of these will get combined.